vendor/pipenv-to-pip

```diff
@@ -7,7 +7,7 @@ import sys
 def main():
     INFILE = sys.argv[1]
 
-    with open(INFILE, 'rb') as f:
+    with open(INFILE, 'r') as f:
```

Comment:
This is an excellent catch. I think a slightly better solution might be to use `io.open` with an explicit encoding to turn it into text that `json.load` could then parse. It would look like:

```py
import io
# ...

def main():
    INFILE = sys.argv[1]

    with io.open(INFILE, 'r', encoding='utf-8') as f:
        lockfile = json.load(f)

# ...
```

Reply:
Agreed. This will ensure that `2.7.x` compatibility stays intact.